tests/test_backend.py

```diff
@@ -3176,16 +3176,16 @@ def func(x, y):
             return tf.cast(s_, tf.float32, name=_TFOUTPUT)
         self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT1: y_val})
 
-    @check_opset_min_version(11)
+    @check_opset_min_version(12)
```

Comment:
Should be 11 instead? Both 11 and 12 opset tests should work (using different implementations).

Reply:
Done.